server/src/user/messages.rs

```diff
@@ -16,7 +16,7 @@ pub enum UserMessage {
     // See https://github.com/serlo/api.serlo.org/issues/459
     ActivityByTypeQuery(user_activity_by_type_query::Payload),
     UserActivityByTypeQuery(user_activity_by_type_query::Payload),
-    UserAddRoleMutation(user_add_role_mutation::Payload),
+    UserAddRoleMutation(user_add_role::Payload),
```

Comment:
```suggestion
    UserAddRoleMutation(user_add_role_mutation::Payload),
```
We name the messages and modules (in messages.rs) after what they are, mutation or query. The functions in model.rs are named after what they do (e.g. add_role).